modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectRelationship/SelectRelationship.tsx

```diff
@@ -25,15 +25,15 @@ export default function SelectRelationship({
 	const [fields, setFields] = useState<ObjectField[]>([]);
 	const options = useMemo(
 		() =>
-			fields.map(
-				({label}) => label[Liferay.ThemeDisplay.getDefaultLanguageId()]!
-			),
+			fields.map(({label}) => {
+				return {
+					label: label[Liferay.ThemeDisplay.getDefaultLanguageId()]!,
+				};
```

Comment:
Hey @naraandrade, to avoid repeating `label[Liferay.ThemeDisplay.getDefaultLanguageId()]` we can instantiate a `const` outside the `SelectRelationship` component and use it in both cases, here and on line `67`, like:

```ts
const defaultLanguageId = Liferay.ThemeDisplay.getDefaultLanguageId();
```
We use this in many other files in the `object-web` and `object-js-components-web` modules to prevent this `Liferay.ThemeDisplay.getDefaultLanguageId()` from getting into the React render stream unnecessarily